sycl-jit/jit-compiler/include/Options.h

```diff
@@ -90,6 +96,17 @@ struct JITTargetInfo
   using OptionBase::OptionBase;
 };
 
+struct JITTargetCPU
+    : public OptionBase<JITTargetCPU, OptionID::TargetCPU, std::string> {
+  using OptionBase::OptionBase;
+};
+
+struct JITTargetFeatures
+    : public OptionBase<JITTargetFeatures, OptionID::TargetFeatures,
+                        std::string> {
+  using OptionBase::OptionBase;
+};
```

Comment:
I'm not sure this `std::string` usage is safe here (linking SYCL and SYCL-JIT with different STL lib versions). Can we use the `DynArray<char>` instead?

Reply:
Good shout, [done](https://github.com/intel/llvm/pull/14860/commits/86f92194f8a64bf06e8782b057138e9944780e7c).